qiskit_nature/second_q/drivers/__init__.py

```diff
@@ -15,6 +15,76 @@
 =============================================================
 
 .. currentmodule:: qiskit_nature.second_q.drivers
+
+Driver Base Class
+=================
+.. autosummary::
+   :toctree: ../stubs/
+   :nosignatures:
+
+   BaseDriver
+   VibrationalStructureDriver
+   ElectronicStructureDriver
+
+Driver Common
+=============
+.. autosummary::
+   :toctree: ../stubs/
+   :nosignatures:
+
+   MethodType
+   InitialGuess
+
+Drivers
+=======
+
+The drivers in the chemistry module obtain their information from classical ab-initio programs
+or libraries. Several drivers, interfacing to common programs and libraries, are
+available. To use the driver its dependent program/library must be installed. See
+the relevant installation instructions below for your program/library that you intend
+to use.
+
+.. toctree::
+   :maxdepth: 1
+
+   qiskit_nature.second_q.drivers.gaussiand
+   qiskit_nature.second_q.drivers.psi4d
+   qiskit_nature.second_q.drivers.pyscfd
+
+Electronic Structure Drivers
+============================
+
+.. autosummary::
+   :toctree: ../stubs/
+   :nosignatures:
+
+   GaussianDriver
+   Psi4Driver
+   PySCFDriver
+
+Vibrational Structure Drivers
+=============================
+
+.. autosummary::
+   :toctree: ../stubs/
+   :nosignatures:
+
+   GaussianForcesDriver
+
+General Driver
+==============
+
+The :class:`GaussianLogDriver` allows an arbitrary Gaussian Job Control File to be run and
+return a :class:`GaussianLogResult` containing the log as well as ready access certain data
+of interest that is parsed from the log.
+
+.. autosummary::
+   :toctree: ../stubs/
+   :nosignatures:
+
+   GaussianLogDriver
+   GaussianLogResult
```

Comment:
The GaussianLogDriver provides the complete text log in the result so you can parse out of it what you want. Now it also has a pre-defined method to parse out the data from that log that is needed for the vibrational, and this is just used by the GaussianForcesDriver - which uses the log driver, to get what it needs from the log result. You do not have to use the vibrational parsing and its not called by the log driver. It was just kept with the result since it acted on the result but could arguably have been parsing logic put elsewhere. People often wanted to use different data from the log, this just gave a way of doing that, and also a specific driver using it - the forces one using it for which there was the pre-made parser method for that data.

Reply:
Thanks for the inside! That actually makes sense, I did not think of that.
Let's leave this text as is then :+1: